libs/unstyled-lfp/src/test/UnstyledLanguageFontPicker.stories.ts

```diff
@@ -11,9 +13,24 @@ export default meta;
 
 type Story = StoryObj<typeof UnstyledLanguageFontPicker>;
 
-export const ExtraFonts: Story = {
+export const MultiselectOff: Story = {
```

Comment:
nit: const objects would typically be camelCase (or SNAKE_CASE but that is more typical for const primatives) to distinguish them from types (PascalCase). Here and below. Perhaps there are different conventions in StoryBook.

Reply:
Yes, the Storybook convention is to have PascalCase for stories (e.g., in https://storybook.js.org/docs/get-started/whats-a-story). I'm just thinking of them on par with React components.